doc/conda/Jenkinsfile.conda.template

```diff
@@ -0,0 +1,82 @@
+pipeline {
```

Comment:
Please, add comments at the head of the file with things that need to be replaced.... might be useful to convert those things into environment variables...